docs/source/tutorials/outliers.ipynb

```diff
@@ -516,10 +544,105 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "937c7e97",
+   "metadata": {},
+   "source": [
+    "## 5. Use cleanlab and `pred_probs` to find outliers in the dataset\n",
+    "\n",
+    "\n",
+    "Another way we can detect unusual exampels in the data is using prediction probabilities for out-of-distribution detection. Instead of using intermediate feature representations, this type of outlier detection simply needs a model's `pred_probs` to work. These methods are faster than using `feature_embeddings` but as a tradeoff perform slightly worse. This is due to the intermediate feature embeddings containing more auxillary information for every example, only some of which is represented in the final predicted probabilites.\n",
+    "\n",
+    "To get `pred_probs` for this example, a Logistic Regression Classifier is fit on the already generated `train_feature_embeddings` from the outlier example above."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "740a8774",
+   "id": "40fed4ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Preprocess data\n",
+    "train_labels = np.array(train_data.dataset.targets)[train_data.indices]\n",
+    "train_labels = np.unique(train_labels, return_inverse=True)[1] # Zero index out train labels\n",
+    "test_labels = np.array(test_data.dataset.targets)[test_data.indices]\n",
+    "\n",
+    "scaler = preprocessing.StandardScaler().fit(train_feature_embeddings)\n",
+    "train_feature_embeddings_scaled = scaler.transform(train_feature_embeddings)\n",
+    "test_feature_embeddings_scaled = scaler.transform(test_feature_embeddings)\n",
+    "\n",
+    "# Create model and fit on scaled features and zero indexed train labels (may take a few minutes)\n",
+    "model = LogisticRegression(penalty='l2', C=1, solver='sag', max_iter=500)\n",
```

Comment:
sorry I overlooked this before, but here we should wrap the LogisticRegression with `sklearn.model_selection.cross_val_predict` as done in our other tutorials:  https://docs.cleanlab.ai/v2.0.0/tutorials/tabular.html

and explain why it's important to use held-out pred_probs. For explanation, you can just link our [cross-validation tutorial](https://docs.cleanlab.ai/v2.0.0/tutorials/pred_probs_cross_val.html), but be sure to format a proper relative rather than absolute link since this needs work for all versions of cleanlab (hopefully you can find an example in existing codebase).

Reply:
That is also important for the accuracy evaluation below to be meaningful at all.

Reply:
To later get predictions for the test data, you can either: retrain another LogisticRegression model on the full training set, or average the pred_probs from each fold's LogisticRegression that was produced during cross-validation.

Reply:
Decided to retrain for simplicity!